### PR TITLE
Content blocker global setting and tutorial

### PR DESCRIPTION
**Pull request by subsymbolic**
Reviewer: Chris
Asana Tutorial page: https://app.asana.com/0/361428290920652/391317240559345
Global on/off setting: https://app.asana.com/0/361428290920652/407600612467832
Disabled popout: https://app.asana.com/0/361428290920652/407600612467833

**Description**:
Separate content blocker global settings from main panel and show content blocker status in omnibar:
- Move global content blocker toggle to settings
- Highlight when content blocker is disabled in omnibar
- Present separate content blocker panel when disabled to encourage user to reenable
- Add mini tutorial to content blocker panel

**Steps to test this PR**:

**TEST 1**
1. Got to settings and enable the content blocker
2. Browse to a page
3. Ensure that the omnibar shows a site rating
4. Press on the site rating, ensure that the ssl, tracker information and new tutorial appear in the content blocker popout

**TEST 2**
1. Got to settings and disable the content blocker
2. Browse to a page
3. Ensure that the content blocker button is a disabled button (red background with exclamation mark "!")
4. Press the disabled button to show a new panel which allows the content blocker to be reenabled
5. Press the toggle to reenable the content blocker and ensure that the std content blocker panel is displayed

**TEST 3**
1. While browsing, visit the settings page and toggle the content blocker button
2. Ensure the correct button is displayed in the omnibar
3. Now repeat step one (toggling the button again in the other direction) and two

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR DRI Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications (Database connections, Grafana stats, CPU)

### CODE REVIEW

**Comment by brindy** [comment]
More of UI/UX thing, but I don't like how it's split across three lines - it looks "off" in some way.  

If you remove the carriage returns so that it wraps and drop the font size down to 14 it looks a lot better imho.

File: DuckDuckGo/ContentBlockerViewController.swift [hunk 1]
```diff
@@ -27,8 +27,8 @@ class ContentBlockerViewController: UITableViewController {
     @IBOutlet weak var httpsLabel: UILabel!
     @IBOutlet weak var blockCountCircle: UIImageView!
     @IBOutlet weak var blockCount: UILabel!
-    @IBOutlet weak var blockingEnabledToggle: UISwitch!
     @IBOutlet weak var blockThisDomainToggle: UISwitch!
+    @IBOutlet weak var tutorialText: UILabel!
```
Comment (brindy) [inline DuckDuckGo/ContentBlockerViewController.swift]: There's a bit of duplication here and in the `ContentBlockerDisabledViewController`: a couple of lines of code in each view controller, but also some UI in the story board.  Could extract the tutorial in to a view controller of its own and embed it on each of the popups instead.

Comment (subsymbolic) [inline DuckDuckGo/ContentBlockerViewController.swift]: Good call, this is a UITableView so I've used a custom UITableViewCell class . I sat on the fence about this as it's more code to get reuse in than just repeating it, but I think the codebase is better for it - repetition is a maintenance nightmare.

Comment (brindy) [inline DuckDuckGo/ContentBlockerViewController.swift]: And repetition in the UI is the worst, would rather have a few more lines of code, tbh.  Good call with the UITableViewCell, that makes sense.

**Comment by subsymbolic** [comment]
Thanks for the review @brindy :)

**Pull request merged**